modules/op-installing-tekton-hub.adoc

```diff
@@ -0,0 +1,98 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-tekton-hub-with-openshift-pipelines.adoc
+
+:_content-type: PROCEDURE
+[id="installing-tekton-hub_{context}"]
+= Installing {tekton-hub}
+
+[role="_abstract"]
+{tekton-hub} is an optional component; cluster administrators cannot install it using the `TektonConfig` custom resource (CR). To install and manage {tekton-hub}, use the `TektonHub` CR.
+
+[discrete]
+.Prerequisites
+* Ensure that the {pipelines-title} Operator is installed in the default `openshift-pipelines` namespace on the cluster.
+
+[discrete]
+.Procedure
+
+. Create a fork of the link:https://github.com/tektoncd/hub[Tekton Hub] repository.
+
+. Clone the forked repository. 
+
+. Update the `config.yaml` file to include at least one user with the following permissions:
+* refresh a catalog
+* refresh a config file
+* create an agent token. 
++
+[source,yaml]
+----
+...
+scopes:
+- name: agent:create
+  users: <your_GitHub_username> <1>
+- name: catalog:refresh
+  users: <your_GitHub_username> <2>
+- name: config:refresh
+  users: <your_GitHub_username> <3>
+...
+----
+<1> An user with `agent:create` scope can set up a cron job that refreshes the {tekton-hub} database after an interval, if there are any changes in the catalog.
+<2> An user with `catalog:refresh` scope can refresh the catalog and all resources in the database of the {tekton-hub}.
+<3> An user with `config:refresh` scope can get additional scopes.
+
+. Commit and push the changes to your forked repository.
+
+. Create an OAuth application with your Git repository hosting provider, and note the Client ID and Client Secret. The supported service providers are GitHub, GitLab, and BitBucket.
+
+. Create the necessary formation for the {tekton-hub} API secret, which contains the following fields.
```

Comment:
Does the user know what is necessary and what isn't? Please provide the information they need to make that determination. Or, if you mean that "a formation" IS necessary, then simply say "Create a formation for the {tekton-hub}..."

Also, the word "formation" seems out of place here. Is there a better/more conventional term to indicate what you mean?